feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -103,6 +105,16 @@ class BillingWrapper(
         }
     }
 
+    private fun isProductDetailsUnupported(): Boolean {
+        if (billingClient == null) {
+            return false
```

Comment:
I'm not sure if we should default this to false... I'm thinking the billing client should always be connected when we call this, so this would be an unexpected situation we should log.